rs/cross-chain/proposal-cli/src/git/mod.rs

```diff
@@ -102,10 +102,11 @@ impl GitRepository {
         from: &GitCommitHash,
         to: &GitCommitHash,
     ) -> ReleaseNotes {
+        const FORMAT_PARAMS: &str = "%C(auto) %h %s";
         let mut git_log = self.git();
         git_log
             .arg("log")
-            .arg("--format=%C(auto) %h %s")
+            .arg(format!("--format={}", FORMAT_PARAMS))
```

Comment:
I couldn't just add the quotes directly here, since then they would be passed down to the actually run command, resulting in them appearing in the list of commits.

Reply:
nice trick!